fluss-client/src/test/java/com/alibaba/fluss/client/write/RecordAccumulatorTest.java

```diff
@@ -391,6 +392,41 @@ void testAwaitFlushComplete() throws Exception {
         assertThatThrownBy(accum::awaitFlushCompletion).isInstanceOf(InterruptedException.class);
     }
 
+    @Test
+    void testReadyWithTimeOut() throws Exception {
```

Comment:
What's the purpose of this test? Do we need to check the `accumulator.ready` result? 

Maybe we can take Kafka's `RecordAccumulatorTest#testNextReadyCheckDelay` as an example.